contrib/examples/walkthrough/ups-instance.yaml

```diff
@@ -7,6 +7,7 @@ spec:
   serviceClassName: user-provided-service
   planName: default
   parameters:
-    credentials:
-      name: root
-      password: letmein
+    inline:
+      credentials:
+        name: root
+        password: letmein
```

Comment:
@duglin just for the clarity: you surely can add support for multiple sources of data for parameters (but it's a separate issue to be designed and addressed in a separate PR):
```
spec:
  serviceClassName: user-provided-service
  planName: default
  parameters:
    instance:
      credentials:
        name: root
        password: letmein
    secretRef:
      name: mySecret
    secretKeyRef:
      name: myOtherSecret
      key: key1
```
I am strongly against any special prefixes like `k8s_*`
The merging strategy is a separate issue, but I would support merging only at the top level.